sendrecv/gst/webrtc-sendrecv.py

```diff
@@ -113,7 +133,11 @@ def start_pipeline(self):
         self.webrtc.connect('pad-added', self.on_incoming_stream)
         self.pipe.set_state(Gst.State.PLAYING)
 
-    async def handle_sdp(self, message):
+        self.bus = self.pipe.get_bus()
+        pollfd = self.bus.get_pollfd()
```

Comment:
Second user of the new API I added in 1.14 :)

Reply:
yes, I was pretty happy to see this had just been added, however I had to fix an annotation issue right after 1.14.2 was merged, which reminds me this will break unless master is used :(

Reply:
I could put this in a try except block for now?